server.js

```diff
@@ -38,16 +38,16 @@ var PATH = 'http://api.openweathermap.org/data/2.5/weather';
                   }).on('end', function() {
                      resbody = Buffer.concat(resbody);
                      //console.log(`Can we print out a raw JSON property? : ${resbody.id}`);// no we can't. It's undefined.
-                     var jsn = JSON.parse(resbody);
+                     const jsn = JSON.parse(resbody);
                      console.log(jsn);
                      console.log(`The weather in ${jsn.name} is ${jsn.weather[0].description}. It\'s 
                            currently ${jsn.main['temp']} degrees K.`);
                         //console.log(typeof jsn.id); // it's a number! That's why we need to toString() it below
 
                      response.writeHead(200, {"Content-Type": "text/plain"})
                      // you can only write back strings or buffers!!!;
-                     response.end(`${jsn.name.toString()}, ${jsn.sys.country.toString()}:
-                            ${jsn.weather[0].description.toString()} and ${jsn.main.temp.toString()} degrees K`);
+                      response.end(`${jsn.name.toString()}, ${jsn.sys.country.toString()}:
+                            ${jsn.weather[0].description} and ${jsn.main.temp.toString()} degrees K`);
```

Comment:
`jsn.weather[0].description` should already be a string as opposed to `jsn.main.temp`, so we shouldn't need to convert it with `.toString()`, right?